examples/interacting_with_js_arrays/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+#[macro_use]
+extern crate neon;
+
+use neon::vm::{Call, JsResult};
+use neon::js::{Variant, JsNumber, JsArray};
+
+// https://github.com/dfrankland/node-eyeliner/blob/master/native/src/js_object_utils.rs
+pub fn js_value_to_float(js_number: Variant) -> f64 {
+  match js_number {
+    Variant::Number(number) => number.value(),
+    _ => panic!("heck"),
+  }
+}
+
+fn accepts_js_arrays(call: Call) -> JsResult<JsNumber> {
+  let js_arr_handle = call.arguments.get(call.scope, 0).unwrap();
```

Comment:
Again, let's not recommend `.unwrap()`. It's better style and more concise to use `?`:

```rust
let js_arr_handle = call.arguments.get(call.scope, 0)?;
```